ios/MaasCore/Sources/MaasCore/Api/ApiError.swift

```diff
@@ -1,8 +1,20 @@
+/**
+ - Tag: ApiError
+ */
 enum ApiError: Swift.Error, Equatable, Hashable {
```

Comment:
`ApiError` should be `public enum` because we won't be able to access it when we install SPM dependencies in to custom project.

Reply:
OK, unrelated to the scope of this PR